### ISSUE TITLE
Mining got stuck after error - yay! found 2 shares at once

### ISSUE THREAD

**Issue by Magadan84** [andru-kun/wildrig-multi#312]
I faced a few times an issue when mining got stuck but at the same time, power consumption and hash rate were shown as normal mining. But only by analyzing a pool statistic it gets understandable that no mining there for a while. Only restarting helps to continue real mining.
The following message is in the miner log when no mining on the pool.

GPU #0: yay! found 2 shares at once
-- 1:wildrig-multi -- time-stamp -- Dec/06/24  8:15:04 --
-- 1:wildrig-multi -- time-stamp -- Dec/06/24  8:25:22 --
[08:25:22] GPU #1: yay! found 2 shares at once

More details are attached.

wildrig-multi version - 0.41.4
HiveOS - 0.6-227@241127
Radeon RX 6800 XT x2
Pool - eu.evrpool.com:1111
Overclock - Pwr   Eff CClk MClk | 157W 0.183 1600 1054
[wildrig issue log.txt](https://github.com/user-attachments/files/18038825/wildrig.issue.log.txt)
![wildrig issue](https://github.com/user-attachments/assets/e6e28ed1-6cf4-41fa-962d-de4d5979e2c1)

**Comment by Magadan84** [comment]
when it crashes it should restart ideally. An alternative option is to show real hashrate = 0, and power consumption needs to be minimal/idle, doesn't it? I have never seen such a wrong representation of crashes and statistics.

**Comment by Magadan84** [comment]
> it should, but I have no idea why it continue to show hashrate while reasults are incorrect(all that "yay! found 2 shares" without real shares found and sent to pool)

Don't rush to answer and draw conclusions, I believe that you can solve this issue or enhance integration with hiveos someday.
Just a kind piece of advice, fee motivates developers to improve the quality, and active issues, especially critical ones, leading to loss of money, motivate users to change a toolset. Good luck.

**Comment by andru-kun** [comment]
> Don't rush to answer and draw conclusions, I believe that you can solve this issue or enhance integration with hiveos someday.
> Just a kind piece of advice, fee motivates developers to improve the quality, and active issues, especially critical ones, leading to loss of money, motivate users to change a toolset. Good luck.

Nah, it doesn't motivate, I develop my miner mostly "just for fun", and lately no more fun investigate AMD problems or something :) Anyway, thanks for the report, some day will check that closely. So far better to use different miner in your case.
